### PR TITLE
ENG-2959 Moving lib versions to parent

### PR DESCRIPTION
**Pull request by BrenoQVDS**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -244,6 +241,10 @@
         <dependency>
             <groupId>io.fabric8</groupId>
             <artifactId>kubernetes-server-mock</artifactId>
+        </dependency>
+		<dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk15on</artifactId>
         </dependency>
```
Comment (firegloves) [inline pom.xml]: Why do we need this? Is it a fabric8 dependency? I'm quite sure that we don't use it in the CM, could it be an option to cut it off the project?

Comment (BrenoQVDS) [reply pom.xml]: You are totally right, I have no idea why I ended up adding it, probably a wrong copy/paste. I just removed it and it runs locally as it should.

**Review by firegloves — APPROVED** [review]
LGTM

**Pull request merged**